src/main/java/com/nitnelave/CreeperHeal/block/CreeperChest.java

```diff
@@ -37,6 +39,7 @@
     {
         super(blockState);
         chest = getBlock();
+        data = chest.getData();
         Inventory inv = ((InventoryHolder) blockState).getInventory();
```

Comment:
Please check before casting, otherwise you can't know that you removed the issue.